argoverse/sensor_dataset_config.py

```diff
@@ -49,8 +47,27 @@ class SensorDatasetConfig:
     camera_sensors: SensorSuiteConfig = MISSING
 
 
-DATASET_NAME = "argoverse-v1.1"
```

Comment:
could we put this in the docstring, instead of removing it completely?

Reply:
Could we move it to the `dataset_name` attribute on line 58?

Reply:
so i didnt quite follow why we need the dataset name on line 58